File: backend/src/main/java/es/uv/etse/twcam/backend/business/Usuario.java
package es.uv.etse.twcam.backend.business;

public class Usuario {
    String user;
    String password;
    
    public boolean login(String user, String password){
        if(user == "carolina" && password == "123abc"){
            return true;
        }else{
            return false;
        }
    }

    public String registrarse(String nombre, String apellido, String dni, 
                                String user, String password ){
                                    
        return "Registro Exitoso";
                                }
}

File: frontend/src/app/compartido/usuario.ts
export class Usuario {
    dni:string;
    nombre:string;
    apellido:string;
    telefono:string;
    correo:string;
    password:string;
    rol:string;
    
    constructor(){
        this.dni = "";
        this.nombre = "";
        this.apellido = "";
        this.telefono = "";
        this.correo = "";
        this.password = "";
        this.rol= "";
    }
}